docs/pages/serving_a_cache.mdx

```diff
@@ -0,0 +1,107 @@
+# Serve a Pelican Cache
+
+A [Pelican](http://pelicanplatform.org/) *Cache* creates a cache that connects to a Pelican data federation to allow faster data access. It stores data that is accessed via a pelican origin in a cache until it gets evicted. The cache has the potential to both be faster and physically closer to your location than your origin. This allows for faster data access when running complex workflows.
+
+This document contains instructions on how to serve a Pelican cache.
+
+## Before Starting
+
+### Install Pelican
+
+If you haven't installed Pelican, follow the instructions to [install pelican](/install.mdx).
+
+For _Linux_ users, it is recommended to install Pelican using one of the package managers (RPM, APK, Deb, etc.) so that Pelican dependencies are automatically handled. You may also run a [Pelican docker image](./install/docker.mdx) to serve a Pelican cache.
+If you prefer to install Pelican as a standalone binary, you need to follow [additional instructions](https://osg-htc.org/docs/data/xrootd/install-standalone/#install-xrootd-standalone) to install dependencies for the Pelican cache.
+
+> Note that serving a Pelican cache with a standalone Pelican binary is possible, but not recommended or supported.
+
+For _macOS_ and _Windows_ users who want to serve a Pelican cache, please use [Pelican docker image](./install/docker.mdx).
+
+### Open Firewall Port for Pelican Cache
+
+The Pelican cache listens to two TCP ports for file transfers and Web UI. By default, the file transfer port is at `8442` and the Web UI and APIs port is at `8444`. If your server has firewall policy in place, please open the two ports for both incoming the outgoing TCP requests to allow the Pelican cache to function as expected.
+
+You may change the port numbers through the [configuration file](./parameters.mdx) with parameter [`Cache.Port`](./parameters.mdx#Cache-Port) and [`Server.WebPort`](./parameters.mdx#Server-WebPort) respectively.
+
+> If it is not possible for you to expose any port through the firewall, Pelican has a special feature called _Connection Broker_, where it allows you to serve a Pelican cache without a public-accessible port and any TLS credential files in place. However, this is an experimental feature and requires the Pelican federation you are joining to be compatible. If you are interested in learning more about _Connection Broker_, please contact help@pelicanplatform.org for further instructions.
+
+### Prepare TLS Credentials
```

Comment:
Can you point this to the related origin documentation instead? so that we don't have duplicated pieces. Like we did here: https://docs.pelicanplatform.org/serving_a_federation#prepare-tls-credentials